test/utils/base/cluster_test_runner.go

```diff
@@ -44,6 +44,8 @@ type ClusterTestRunnerBase struct {
 	unitTestMock      bool
 }
 
+var _ ClusterTestRunner = &ClusterTestRunnerBase{}
```

Comment:
Any particular reason for doing it?

Reply:
To force a compilation error if the ClusterTestRunnerBase type does not implements the ClusterTestRunner interface....
So, instead of you reviewing and discovering I missed to update the interface I would get a compilation error in the pipeline.

https://splice.com/blog/golang-verify-type-implements-interface-compile-time/

Reply:
Ok, yes it makes sense, but maybe another approach would be to change the following declaration in the unit test to:
```
func TestGetContext(t *testing.T) {
	var c ClusterTestRunner = &ClusterTestRunnerBase{}
```

Reply:
Okey!

Reply:
nop, actually it does not work, I need access to the object internals for the test

Reply:
It does, but will require type assertions when setting the internal fields.

Reply:
mmm yup, that could work... as I understand it, the test is testing the base type. I prefer to keep the test agnostic to the interface...

Reply:
Ok. Not a big deal. Approving.